atomicapp/install.py

```diff
@@ -172,7 +172,8 @@ def _installDependencies(self):
                 printStatus("Pulling %s ..." % image_name)
                 component_app = Install(self.nulecule_base.answers_data, image_name, self.nulecule_base.nodeps, 
                                         self.nulecule_base.update, component_path, self.dryrun)
-                values = Utils.update(values, component_app.install())
```

Comment:
@goern 
The issue was with line
I have made the change so that instaed of returning values from install() we get it from a field called answers_file_values